src/webview/java/org/medicmobile/webapp/mobile/MedicAndroidJavascript.java

```diff
@@ -212,6 +215,21 @@ public void sms_send(String id, String destination, String content) throws Excep
 		}
 	}
 
+	@android.webkit.JavascriptInterface
+	public void cht_launchExternalApp(String action, String category, String type, String extras, String uri, String packageName, String flags) {
```

Comment:
I don't think we need the `cht_` prefix here, and it looks weird to mix and match underscores and camel case.

```suggestion
	public void launchExternalApp(String action, String category, String type, String extras, String uri, String packageName, String flags) {
```

Reply:
Done